diff --git a/node_modules/react-native-phone-number-input/lib/index.js b/node_modules/react-native-phone-number-input/lib/index.js
index b89c388..3983b42 100644
--- a/node_modules/react-native-phone-number-input/lib/index.js
+++ b/node_modules/react-native-phone-number-input/lib/index.js
@@ -7,7 +7,7 @@ import CountryPicker, {
   CountryModalProvider,
   Flag,
 } from "react-native-country-picker-modal";
-import { PhoneNumberUtil } from "google-libphonenumber";
+import { PhoneNumberUtil, AsYouTypeFormatter } from "google-libphonenumber";
 import styles from "./styles";
 
 const dropDown =
@@ -93,7 +93,15 @@ export default class PhoneInput extends PureComponent {
 
   onChangeText = (text) => {
     this.setState({ number: text });
+    const originalNumber = (text.match(/\d/g) ?? []).join("");
     const { onChangeText, onChangeFormattedText } = this.props;
+    const formatter = new AsYouTypeFormatter(this.state.countryCode || 'US');
+        let formatted = '';
+        formatter.clear();
+        for (let c of originalNumber) {
+          formatted = formatter.inputDigit(c);
+        }
+        this.setState({ number: formatted });
     if (onChangeText) {
       onChangeText(text);
     }
